### PR TITLE
Write JSDoc for many static creation operators

### PR DESCRIPTION
**Pull request by staltz**
empty, never, throw, of, create, defer, range, interval, timer.

### CODE REVIEW

File: spec/observables/empty-spec.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {expect} from 'chai';
```
Comment (blesh) [inline spec/observables/empty-spec.ts]: Looks like this is unused and breaking the build.

File: spec/observables/never-spec.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import {expect} from 'chai';
```
Comment (blesh) [inline spec/observables/never-spec.ts]: again, unused and breaking the build.

**Comment by staltz** [comment]
Yep, we saw the same thing. I patched it.

**Comment by blesh** [comment]
LGTM.

**Comment by blesh** [comment]
Merged with 4b30b15. Thanks @staltz

**Pull request closed without merging**